test/JDBC/upgrade/latest/schedule

```diff
@@ -432,3 +432,4 @@ BABEL_4817
 BABEL-4863
 BABEL-4869
 BABEL-3401
+babel_726
```

Comment:
shouldn't we add babel_726-before-14_12... file entry to older version schedule files?